File: aAlvarezAssignment4/Car.java
package aAlvarezAssignment4;

class Car {
    private String makeAndModel;
    private int maximumNumberOfPassengers;
    private int numberOfDoors;
    Car(String makeAndModel, int maximumNumberOfPassengers, int numberOfDoors){
        this.makeAndModel= makeAndModel;
        this.maximumNumberOfPassengers= maximumNumberOfPassengers;
        this.numberOfDoors= numberOfDoors;
    }
    Car(){
        this.makeAndModel= "not assigned";
        this.maximumNumberOfPassengers= 2;
        this.numberOfDoors= 2;
    }

    public void setMakeAndModel(String m){
        this.makeAndModel= m;
    }

    public String getMakeAndModel(){
        return makeAndModel;
    }

    public void setMaximumNumberOfPassengers(int m){
        this.maximumNumberOfPassengers= m;
    }

    public int getMaximumNumberOfPassengers(){
        return maximumNumberOfPassengers;
    }

    public void setNumberOfDoors(int m){
        this.numberOfDoors= m;
    }

    public int getNumberOfDoors(){
        return numberOfDoors;
    }

    public String toString(){
        return "Make and Model: " + makeAndModel + "\nMaximum Number of Passengers: " + maximumNumberOfPassengers + "\nNumber of Doors: " + numberOfDoors;
    }

    }

File: aAlvarezAssignment3/Car.py
class Car:
    def __init__(self,makeAndModel="none assigned",numberOfDoors=4,maximumNumberOfPassengers=5):
        self.model= makeAndModel
        self.doors= numberOfDoors
        self.passengers= maximumNumberOfPassengers

    def setMakeAndModel(self,make):
        self.model= make

    def getMakeAndModel(self):
        return self.model
    
    def getMaximumNumberOfPassengers(self):
        return self.passengers
    
    def __str__(self):
        return "Make and Model: " + self.model + "\nNumber of Doors: " + str(self.doors) + "\nMaximum Number of Passengers: " + str(self.passengers)
